ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml

```diff
@@ -46,6 +46,9 @@ ref:
   - name: OPENSHIFT_INSTALL_AWS_PUBLIC_ONLY
     default: ""
     documentation: "Whether to use only public subnets for AWS. Implies no NAT Gateways. Requires a VPC to be configured prior to install."
+  - name: ENABLE_EFA_PG_INSTANCE
+    default: "false"
+    documentation:  "Use AWS EFA Network Interface Type and placement group for one worker node.  Set to 'true' to opt into efa and pg instance.  Explicitly set to 'false' to opt out.  Leave unset for the default, which may change."
```

Comment:
Can we mentioned `c5n.9xlarge` instance type and `pgcluster` placementGroupName would be used, once this option is enabled.

Reply:
ok, let me update it.